src/components/DistanceRequest/index.js

```diff
@@ -38,6 +39,9 @@ const propTypes = {
     /** Are we editing an existing distance request, or creating a new one? */
     isEditingRequest: PropTypes.bool,
 
+    /** Are we editing a new distance request */
```

Comment:
Can you add more details to this comment to make it clear what this means? I believe it's editing from the confirmation screen when creating a new request. People might think it means editing the waypoints the first time when creating a new distance request.

Reply:
Nice thanks